### ISSUE TITLE
[HotChocolate] generic inherited fields fails on sorting

### ISSUE THREAD

**Issue by PascalSenn** [ChilliCream/hotchocolate#1611]
**Describe the bug**
When you extend Identity Framework's User you are not longer able to sort by `id` Probably has to do with generic inheritance 

https://github.com/dotnet/aspnetcore/blob/master/src/Identity/Extensions.Stores/src/IdentityUser.cs

```csharp
  public class IdentityUser<TKey> where TKey : IEquatable<TKey>
    { 
   ... 
   
        public virtual TKey Id { get; set; }
```` 

**To Reproduce**
Extend Generic class and sort by ID

**Expected behavior**
Sorting by Id should work
**Desktop (please complete the following information):** 
 - Version 10.4.0

**Additional context**
-

**Comment by PascalSenn** [comment]
https://github.com/ChilliCream/hotchocolate/releases/tag/10.4.1-preview.2

**Comment by michaelstaib** [comment]
Is this fix also in 11?

**Comment by PascalSenn** [comment]
yes

**Comment by sgabler-solytic** [comment]
@PascalSenn Thanks for reacting so quickly when we reported this. I just tested this on 10.4.3, but I still get errors. Just double checking: this fix is in 10.4.3, right?

**Comment by sgabler-solytic** [comment]
Sorry, the sorting actually works in 10.4.3. My error must be something different.

Will investigate some more and maybe open an issue, if needed. Sorry for confusion.
